docs/docs/07-forms.md

```diff
@@ -81,6 +87,8 @@ An `<input>` that does not supply a `value` (or sets it to `null`) is an *uncont
 
 This will render an input that starts off with an empty value. Any user input will be immediately reflected by the rendered element. If you wanted to listen to updates to the value, you could use the `onChange` event just like you can with controlled components.
 
+*Uncontrolled*, means it maintains its own internal state.
```

Comment:
"it" is a little ambiguous.  How about "*Uncontrolled*, means the `<input>` component maintains its own internal state."